File: Publisher Integration/SPX Mediation/Unified Bidding | iOS Integration/Example code for Banners.m
// Setting age and gender 
SmaatoSDK.userGender = kSMAGenderFemale;
SmaatoSDK.userAge = @30;
 
// Let's create and set targeting parameters 
SMAKeyValuePairs *KVP = [SMAKeyValuePairs new];
[KVP setTargetingValue:@"30" forKey:@"Age"]; // replace with your key-value pairs configured on SPX site 
[KVP setTargetingValues:@[@"Demo", @"Stage"] forKey:@"Application"]; // replace with your key-value pairs configured on SPX site   
// Setting KV pairs for Banner request 
SMABannerView.keyValuePairs = KVP;
 
// Prebid request 
[SmaatoSDK prebidBannerForAdSpaceId:@<SMAATO_ADSPACE_ID>"
                         bannerSize:kSMAUbBannerSizeXXLarge_320x50
                         completion:^(SMAUbBid *_Nullable bid, NSError *_Nullable error) {
     //Loading AdMob ad                              
}];

File: Publisher Integration/SPX Mediation/Unified Bidding | iOS Integration/Example code for Banners.swift
// Setting age and gender 
SmaatoSDK.userAge = 35
SmaatoSDK.userGender = .male
   
// Defining Key-value targeting 
let KVP = SMAKeyValuePairs()
KVP.setTargetingValues(["Demo", "Stage"], forKey: "Application") // replace with your key-value pairs configured on SPX site 
KVP.setTargetingValues(["iOS", "Android"], forKey: "OS") // replace with your key-value pairs configured on SPX site  
// Setting KV pairs for Banner request 
SMABannerView.keyValuePairs = KVP
   
// Prebid request 
SmaatoSDK.prebidBanner(forAdSpaceId: "adSpaceId",
                         bannerSize: .xxLarge_320x50) { (bidUnit, error) in
    //Loading MoPub ad
}